lms/djangoapps/verify_student/views.py

```diff
@@ -1350,7 +1350,7 @@ def get(self, request):
         Backbone views used in the initial verification flow.
         """
         status, _ = SoftwareSecurePhotoVerification.user_status(request.user)
-        if status in ["must_reverify", "expired"]:
+        if status in ["must_reverify", "expired", "pending"]:
```

Comment:
Please add comment about the status and their mapping to photo verification statuses i.e. when we get 'pending' status for a photo verification?